Include/pegen_interface.h

```diff
@@ -0,0 +1,32 @@
+#ifndef Py_LIMITED_API
```

Comment:
If the whole content is excluded from the limited C API, you can move the file to Include/cpython/ which means that you have to use `#include "cpython/pegen_interface.h"`.

Reply:
Do we need to have the `#ifndef Py_LIMITED_API` in that case?